MarsMission/getData.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  getData.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/04.
+//
+
+import Foundation
+
+public class GetData {
+    
+    @Published var atmosphericTemperatureData = [AtmosphericTemperatureData]()
+    
+    init() {
+        fecthData()
+    }
+    
+    func fecthData() {
```

Comment:
This fetch data function must contain a completion handler

Reply:
<img width="875" alt="Screenshot 2020-11-05 at 05 52 14" src="https://user-images.githubusercontent.com/55865505/98195799-482e9800-1f2b-11eb-971d-f0dfe729845a.png">